README.md

```diff
@@ -133,6 +136,11 @@ Must be non-nil and non-empty. The structure of the `pipeline` object is as foll
  containing variables to be interpolated via `{{ }}` in `config_file`.
  Equivalent of `-l some-vars-file.yml` in `fly set-pipeline` command.
 
+ - `vars`: *Optional.* Map of keys and values corresponding to variables
+ to be interpolated via `{{ }}` in `config_file`. Values can arbitrary
```

Comment:
This should say `(( ))` instead of `{{ }}` (curly braces are the older, deprecated form).